env.example

```diff
@@ -1,7 +1,8 @@
+DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT=https://kovan-ec.example.com
```

Comment:
This should be removed, yeah?

Reply:
Hmm I don't think so. Without `DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT` the replica won't work, so it needs to be in `.env`.

Reply:
Oh sure, an endpoint is needed. I meant the value. We can leave 
`DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT=`
or something like
`DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT=WONT_WORK_UNLESS_YOU_PROVIDE_A_VALID_L1_ENDPOINT`
I was leaving it out and hoping people would read the docs.

Reply:
Ah right. How about with `DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT=https://AN-ETHEREUM-RPC-ENDPOINT-HERE-OR-IT-WONT-WORK`